src/main/java/uk/ac/ebi/eva/t2d/jobs/steps/ReadSampleDefinitionStepConfiguration.java

```diff
@@ -0,0 +1,75 @@
+/*
+ * Copyright 2017 EMBL - European Bioinformatics Institute
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package uk.ac.ebi.eva.t2d.jobs.steps;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.batch.core.Step;
+import org.springframework.batch.core.configuration.annotation.EnableBatchProcessing;
+import org.springframework.batch.core.configuration.annotation.StepBuilderFactory;
+import org.springframework.batch.core.listener.ExecutionContextPromotionListener;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.context.annotation.Profile;
+import uk.ac.ebi.eva.pipeline.Application;
+import uk.ac.ebi.eva.t2d.jobs.tasklet.ReadSampleDefinitionTasklet;
+import uk.ac.ebi.eva.t2d.parameters.T2dJobParametersNames;
+import uk.ac.ebi.eva.t2d.parameters.T2dTsvParameters;
+
+import static uk.ac.ebi.eva.t2d.BeanNames.T2D_READ_SAMPLE_DEFINITION;
+
+@Configuration
+@Profile(Application.T2D_PROFILE)
+@EnableBatchProcessing
+public class ReadSampleDefinitionStepConfiguration {
+
+    private static final Logger logger = LoggerFactory.getLogger(ReadSampleDefinitionStepConfiguration.class);
+
```

Comment:
Probably can give function name / comment what it does like promoting parameters toJobContext at the end of step